### DISCUSSION TITLE
Customize search result heading

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#5242] [Q&A]
Where would be the right place to tweak the content of the search results? 
In particular, I want to display the full path in addition to the heading of the file

In the example below, I need to have the path (e.g. `oddly/long/path/to/file`) right below the Heading.

<img width="830" alt="image" src="https://user-images.githubusercontent.com/73690588/226950757-2da27db2-b158-4002-a027-07e81cc7cfc5.png">

**Comment by squidfunk** [comment]
Search is rendered in JavaScript, so you would need to make your changes to [this template](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/javascripts/templates/search/index.tsx) and [rebuild the theme](https://squidfunk.github.io/mkdocs-material/customization/#theme-development) 😊

**Reply by ghost** [reply]
Thanks for the ultra-fast response. 

I guess there is no way of getting around rebuilding the theme, right? Not even with an additional `extra_javascript` 🤞? 

Before I commit to such drastically workarounds, I will probably try to get around this with better headings 😁 ... 

But thanks for the help (and the fantastic framework)

**Reply by squidfunk** [reply]
Yes, you'd need to rebuild the theme. All is hackable on Material for MkDocs, but some things need a little more effort. Sorry for that. Note that we'll be revisiting search later this year. Also, possible related: #3787
